tests/integration/plugins/test_attachment.py

```diff
@@ -21,7 +21,7 @@ async def test_attachment_plugin_new_record(
     editor_client = make_client(editor_auth)
     with open("kinto-logo.svg", "rb") as attachment:
         assert requests.post(
-            f"{editor_client.session.server_url}{await editor_client.get_endpoint('record', id='logo')}/attachment",
+            f"{editor_client.session.server_url}{await editor_client.get_endpoint('record', id='logo')}/attachment",  # noqa: E501
```

Comment:
Should we just ignore `E501` globally and let the formater take care of long lines when it can?

Reply:
Oh wait, just noticed https://github.com/mozilla/remote-settings/pull/500/commits/dccb698d3ad0bcac5dc7fc30d723b8dfe5422dd0. What motivated this commit?

Reply:
We had `noqa` strings that ruff could not handle, I thought it was redundant and went for the route of keeping the lint rule, but yeah, let's handle this with format